Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -52,33 +54,44 @@
 
 
 class WebPlatformTestExporter(object):
-    def __init__(self, host, options, gitClass=Git, bugzillaClass=Bugzilla, WPTGitHubClass=WPTGitHub, WPTLinterClass=WPTLinter):
+    def __init__(self, host, options, gitClass=Git, bugzillaClass=Bugzilla, WPTGitHubClass=WPTGitHub, WPTLinterClass=WPTLinter, repository=None):
         self._host = host
         self._filesystem = host.filesystem
         self._options = options
 
         self._host.initialize_scm()
 
         self._WPTGitHubClass = WPTGitHubClass
-        self._gitClass = gitClass
+        self._gitClass = gitClass  # FIXME: Move git operations to webkitscmpy
+        self.repository = repository or local.Git(os.path.abspath(os.getcwd()))
```

Comment:
We use `local.Scm.from_path(self.host.filesystem.getcwd())` elsewhere, interestingly. I don't know how much we care about the filesystem abstraction here…